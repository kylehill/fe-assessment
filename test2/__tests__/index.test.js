import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../pages/index'

describe("Smoketest", () => {
  it("renders the Index page", () => {
    const component = shallow(<Index />)
  })
})

describe("Snapshot", () => {
  it("matches the existing snapshot", () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})