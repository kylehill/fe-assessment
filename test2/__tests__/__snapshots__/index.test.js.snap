// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot matches the existing snapshot 1`] = `
Array [
  <ul
    className="jsx-1976454187 multiselect-list"
  >
    <li
      className="jsx-1384089475 room "
    >
      <header
        className="jsx-1384089475"
      >
        Room 
        1
        <br
          className="jsx-1384089475"
        />
      </header>
      <section
        className="jsx-1384089475"
      >
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Adults (18+)
          </div>
          <select
            className="jsx-4138645028"
            disabled={false}
            onChange={[Function]}
            value={1}
          >
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Children (0-17)
          </div>
          <select
            className="jsx-4138645028"
            disabled={false}
            onChange={[Function]}
            value={0}
          >
            <option
              className="jsx-4138645028"
              value={0}
            >
              0
            </option>
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
      </section>
    </li>
    <li
      className="jsx-1384089475 room room-disabled"
    >
      <header
        className="jsx-1384089475"
      >
        <input
          checked={false}
          className="jsx-1384089475"
          onChange={[Function]}
          type="checkbox"
        />
        Room 
        2
        <br
          className="jsx-1384089475"
        />
      </header>
      <section
        className="jsx-1384089475"
      >
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Adults (18+)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={1}
          >
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Children (0-17)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={0}
          >
            <option
              className="jsx-4138645028"
              value={0}
            >
              0
            </option>
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
      </section>
    </li>
    <li
      className="jsx-1384089475 room room-disabled"
    >
      <header
        className="jsx-1384089475"
      >
        <input
          checked={false}
          className="jsx-1384089475"
          onChange={[Function]}
          type="checkbox"
        />
        Room 
        3
        <br
          className="jsx-1384089475"
        />
      </header>
      <section
        className="jsx-1384089475"
      >
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Adults (18+)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={1}
          >
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Children (0-17)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={0}
          >
            <option
              className="jsx-4138645028"
              value={0}
            >
              0
            </option>
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
      </section>
    </li>
    <li
      className="jsx-1384089475 room room-disabled"
    >
      <header
        className="jsx-1384089475"
      >
        <input
          checked={false}
          className="jsx-1384089475"
          onChange={[Function]}
          type="checkbox"
        />
        Room 
        4
        <br
          className="jsx-1384089475"
        />
      </header>
      <section
        className="jsx-1384089475"
      >
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Adults (18+)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={1}
          >
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
        <div
          className="jsx-4138645028 room-dropdown"
        >
          <div
            className="jsx-4138645028 room-dropdown-title"
          >
            Children (0-17)
          </div>
          <select
            className="jsx-4138645028"
            disabled={true}
            onChange={[Function]}
            value={0}
          >
            <option
              className="jsx-4138645028"
              value={0}
            >
              0
            </option>
            <option
              className="jsx-4138645028"
              value={1}
            >
              1
            </option>
            <option
              className="jsx-4138645028"
              value={2}
            >
              2
            </option>
          </select>
        </div>
      </section>
    </li>
  </ul>,
  <button
    className="jsx-1976454187 multiselect-button"
    onClick={[Function]}
  >
    Submit
  </button>,
]
`;
